.github/workflows/ci.yml

```diff
@@ -55,20 +33,20 @@ jobs:
           discovery.type: single-node
 
     steps:
+    - name: Checkout git repo
+      uses: actions/checkout@v3
```

Comment:
Checking-out the code needs to be the first step now, otherwise we cannot setup python from the `.python-version` file.